cadc-gms/src/intTest/resources/config/cadc-registry.properties

```diff
@@ -7,4 +7,4 @@ ca.nrc.cadc.reg.client.RegistryClient.baseURL = https://haproxy.cadc.dao.nrc.ca/
 ivo://ivoa.net/sso#OpenID = https://ska-iam.stfc.ac.uk/                                                                                       
 
 http://www.opencadc.org/std/posix#user-mapping-0.1 = ivo://opencadc.org/src/posix-mapper
-
+#http://www.opencadc.org/std/posix#user-mapping-0.1 = https://haproxy.cadc.dao.nrc.ca/src/posix-mapper
```

Comment:
Development environment.  It's commented out though so should be fine.

Reply:
yes, bot other parts are just what I happen to have configured for SRCNetso not highly usable by anyone else right now anyway